src/main/java/TransactionApp.java

```diff
@@ -0,0 +1,105 @@
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class TransactionApp {
+
+    private static final TransactionDAO DAO = new TransactionDAO();
+    private Scanner scanner;
+
+    public TransactionApp() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void run() {
+
+        while (true) {
+            printMenu();
+            switch (scanner.nextLine()) {
+                case "1":
+                    addNewTransaction();
+                    break;
+                case "2":
+                    modifyTransaction();
+                    break;
+                case "3":
+                    deleteTransaction();
+                    break;
+                case "4":
+                    displaySumAmount(TransactionType.INCOME);
+                    break;
+                case "5":
+                    displaySumAmount(TransactionType.EXPENSE);
+                    break;
+                case "0":
+                    DAO.close();
+                    return;
+                default:
+                    System.out.println("Podałeś nieprawidłową opcję");
+            }
+        }
+    }
+
+    private void modifyTransaction() {
+        System.out.println("Podaj id transakcji, którą chcesz zmodyfikować");
+        Integer id = scanner.nextInt();
+        scanner.nextLine();
+        Transaction transaction = transactionCreator();
+        transaction.setId(id);
+        DAO.update(transaction);
+    }
+
+
+    private void deleteTransaction() {
+        System.out.println("Podaj id transakcji");
+        Integer id = scanner.nextInt();
+        scanner.nextLine();
+        DAO.delete(id);
+
+    }
+
+    private void addNewTransaction() {
+        DAO.save(transactionCreator());
```

Comment:
Przydałoby się wyświetlić potwierdzenie zapisu obiektu wraz z wygenerowanym przez bazę id. Jeżeli dodamy obiekt, a później chcemy go zaktualizować, to użytkownik jest proszony o id, ale nie wiadomo jakie to id jest